e2e/constants/constants.ts

```diff
@@ -0,0 +1,13 @@
+export const HOST = process.env.HOST!;
+export const TOKEN = process.env.TOKEN!;
+export const FEATURE_TAG = 'javascript'
+export const USER_ID = 'bucketeer-js-user-id-1'
+
+export const FEATURE_ID_BOOLEAN = 'feature-js-e2e-boolean'
+export const FEATURE_ID_STRING = 'feature-js-e2e-string'
+export const FEATURE_ID_INT = 'feature-js-e2e-int'
+export const FEATURE_ID_DOUBLE = 'feature-js-e2e-double'
+export const FEATURE_ID_JSON = 'feature-js-e2e-json'
+
+export const GOAL_ID = 'goal-js-e2e-1'
+export const GOAL_VALUE = 1.0
```

Comment:
I'd like to replace the test data sample here with a set for the node.js SDK

Reply:
It will be better to create new flags for nodejs tests.

Reply:
I updated the nodejs flags